algorithmia/manifest.json

```
@@ -0,0 +1,28 @@
+{
+  "display_name": "Algorithmia",
+  "maintainer": "support@algorithmia.io",
+  "manifest_version": "1.0.0",
+  "name": "algorithmia",
+  "metric_prefix": "algorithmia.",
+  "metric_to_check": "",
```

```suggestion
  "metric_to_check": "algorithmia.duration_milliseconds",
```
We should specify a metric here to ensure that the tile gets automatically installed when this metric is detected.

---

Done in d81ad5f, thanks!